src/content/who/who.json

```diff
@@ -3,7 +3,7 @@
   "img2": "./img2.jpg",
   "img3": "./img3.jpg",
   "title1": "Who we are",
-  "p1": "Sandbox is a proudly student-led software group focusing on the enablement of research through technology. We do this by working directly with PIs to build powerful software to help in experiment conduction and data collection.",
+  "p1": "Sandbox is a club at Northeastern University that accelerates research by building custom software applications to enable data collection, visualization, and more. We work closely with researchers to help them best leverage computation in their work and care deeply about the quality of our software.",
```

Comment:
I think the last sentence is a little run-on, you could just cut out the "and care deeply about the quality of our software" because that's heavily displayed in the next section anyways and our homepage encourages scrolling already so odds are someone would scroll down.